### ISSUE TITLE
Dev test: Wave Interference 1.0.0-dev.66

### ISSUE THREAD

**Issue by samreid** [phetsims/QA#264]
@ariel-phet and @KatieWoe, Wave Interference 1.0.0-dev.66 is ready for the dev testing.  This sim has a publication deadline of Jan 31, 2019 but will be great to complete sooner because other sims share the same deadline.  The dev test should focus on searching for buggy or sim breaking behavior.  This is a PhET Brand sim and does not require PhET-iO testing.

The first round of dev testing was completed in #238.  Since then there have been numerous major changes in the implementation and complex new features added.  This is the second round.  Please be aware some issues discovered during #238 were marked as deferred.

Please document issues in https://github.com/phetsims/wave-interference/issues.  Assign issues to @samreid and link them back to this QA issue.

In the prior issue #238 I was directed to label top priority and assign to Katie.  Not sure whether this issue has the same priority or not, but I'll use that as a starting point.  Co-assigning @ariel-phet so he can confirm scheduling, priority, etc.

<details>
<summary><b>General Dev Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.

<h3>Focus and Special Instructions</h3>

<h3>General Dev Test Platforms</h3>

- [x] Latest macOS, Chrome and Safari
- [x] Latest iOS, Safari
- [x] Windows 10, all browsers
- [x] Latest Chrome OS, Chrome

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/wave-interference/1.0.0-dev.66/phet/wave-interference_en_phet.html)**

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](link).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

UPDATE: I noticed other issues were already marked with top and high priority, so I'll remove the priority label until @ariel-phet can take a look.

**Comment by ariel-phet** [comment]
@KatieWoe marking high priority currently. Once EFAC dev testing (current round) is complete, this sim will likely move to top priority.

**Comment by KatieWoe** [comment]
QA is done

**Comment by samreid** [comment]
Thanks @KatieWoe and the entire QA team!  I'll follow up in the opened issues.
